### DISCUSSION TITLE
Splitting Equisetum?

### DISCUSSION THREAD

**Discussion by MichaelKessler979** [pteridogroup/ppg#84] [General]
One issue I stumbled across recently are the two subgenera of Equisetum. Granted, Equisetum has few species, so a split might not seem necessary for practical purposes, but the two subgenera are morphologically distinct, have different anatomy, chromosome numbers, and do not hybridize with each other (contrary to rampant hybridization within the subgenera). And they have been treated as distinct genera in the past, so no nomenclatural changes are needed, as far as I can tell. And in comparison to other families, where genera are much more finely split, the two subgenera in Equisetaceae seem to be at least as distinct. Any feelings on whether a proposal might make sense?

**Comment by choess** [comment]
Three subgenera, I think; if I recall correctly, _E. bogotense_ is basal to the two main clades of the genus, and was accommodated in the monotypic subgenus _Paramochaete_ by Christenhusz et al. 2019.

I think the subgenera of Equisetum are certainly defensible as genera, and have been used as such on and off (although I don't think it's been common to do so since Hauke's treatment in the 1960s). On the other hand, I'm not sure the advantages to be gained from the new circumscription justify the change. (New combinations would also be needed for _E. pyramidale_, _E. bogotense_, and some of the hybrids.)

As a general principle, I am a bit concerned about how copious changes in circumscription will affect the acceptance of PPG II by a wider botanical audience. There are certainly cases where these are convincing, and we have the phylogenetic data to back them, but in situations like this that are more a matter of taste, I think I would prefer to maintain the current circumscription. If the broader community finds the subgenera useful over time and regularly refers to them (e.g., the way Amauropelta etc. were recognized in Thelypteris historically) that would support the idea of splitting in a future classification.

**Comment by Libor-Ekrt** [comment]
I totally agree with Michael. The division of the existing genus Equisetum into two (-three) equal-valued genera Equisetum, Hippochaete (-Paramochaete) makes sense to me. The groups are really quite different, besides what Michael mentions so there is a big difference in overall genome size between the main (sub)genera. The split would also make sense when compared to the other fern groups we are splitting. I wouldn't worry about negative acceptance by the wider community. On the contrary, I, for example, am constantly asked by students and colleagues why these types are not divided as genera when they are so different and we still call it Equisetum. I am clearly leaning towards a split.

**Comment by SchneiderHarald** [comment]
Dear All, despite I accept the two groups are obviously a bit different and in the light of the complex taxonomic history, I am not in favor to propose two genera. Firstly, the phylogenetic ambiguities have not been fully resolved, e.g. placement of Equisetum bogotense. Secondly, I am not in favor to use the argument that we may oversplit some groups to defend not very helpful segregations. In turn, we may want to be a bit more conservative about proposing new generic names instead of utilizing more frequently the fully capacities of hierarchical classifications, e.g. subgenera and tribes. Finally, I am a bit worried about the expansion to the fossil record. The name Equisetum has been always used in a kind of s.l. Definition in studies requiring placing the Mesozoic and Cenozoic Fossils of horsetails.

**Comment by joelnitta** [comment]
This was voted on but did not pass: https://github.com/pteridogroup/ppg/issues/112
